source/includes/_build.md

```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
 
 ## Linux or macOS
 
-For Iroha, a provisioning of Postgres and Redis are required. You may launch them on your local machine, or use docker containers, as provided.
-
 > Launching Docker and Postgres in Docker
 
 ``` shell
-docker run --name some-redis -p 6379:6379 -d redis:3.2.8
-docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres:9.5
+docker run --name some-redis \
```

Comment:
May be not to split this lines? They are quite short.

Reply:
I expanded the size of text section to 35% for better readability and this is the reason of splitting it

Reply:
<img width="489" alt="screen shot 2017-12-16 at 16 22 36" src="https://user-images.githubusercontent.com/11841667/34070802-624d9f58-e27d-11e7-976f-01440baa4e84.png">

Reply:
Now it look like this, I also changed the password to work with tests and example of config:
![screen shot 2017-12-16 at 16 35 24](https://user-images.githubusercontent.com/11841667/34070945-37747214-e27f-11e7-89de-da043d904cb2.png)